### PR TITLE
Add ocetrac

### PR DESCRIPTION
**Pull request by hscannell**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/ocetrac/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - pytest
+    - numpy
+    - xarray
+    - dask
+    - scipy
+    - scikit-image
+
+test:
+  requires:
+    - pytest
```
Comment (xhochy) [inline recipes/ocetrac/meta.yaml:33]: We only do an import test, so we don't need `pytest` but we should check pip consistency.
```suggestion
  requires:
    - pip
  commands:
    - pip check
```

**Comment by hscannell** [comment]
@jbusecke would you mind taking a look at this? I'm getting an `exit code 1` error that I'm not quite sure about.

**Comment by hscannell** [comment]
The `sha256` key was incorrect since I made a new release to PyPI. Fixed it and checks pass now.

**Comment by hscannell** [comment]
@conda-forge-admin please accept PR. Thanks!

File: recipes/ocetrac/meta.yaml (hunk 1 above)
Comment (hscannell) [reply recipes/ocetrac/meta.yaml]: Thank you @xhochy! I've made your changes.

File: recipes/ocetrac/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  number: 0
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed .
+  
+requirements:
+  build:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - pytest
```
Comment (dopplershift) [inline recipes/ocetrac/meta.yaml:24]: I don't think `pytest` is actually needed to use `ocetrac`?

Comment (hscannell) [reply recipes/ocetrac/meta.yaml:24]: It's used in the tests, but you're right, you don't need it to use `ocetrac`.

**Comment by jbusecke** [comment]
Seems like I saw this too late? Everything good now?

**Review by dopplershift — APPROVED** [review]

**Pull request merged**